src/components/map/readme.md

```diff
@@ -29,13 +29,15 @@ Example: `-34.87945241095968, -56.078210142066956`
+| --------------------- | ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | ---------------------- |
+| `center`              | `center`                | The coord of initial center of the map.                                                                                                                                                                                      | `string`                                | `"0, 0"`               |
+| `highAccuracyLocator` | `high-accuracy-locator` | Enable the High Accuracy in user location. _Note: Set this when `watchPosition = true`._                                                                                                                                     | `boolean`                               | `true`                 |
+| `mapProvider`         | `map-provider`          | The map provider. _Note: Currently, this property is for setting a custom map provider using an URL._                                                                                                                        | `string`                                | `undefined`            |
+| `mapType`             | `map-type`              | Map type to be used. _Note: If you set a map provider, the selected map type will be ignored._                                                                                                                               | `"hybrid" \| "satellite" \| "standard"` | `"standard"`           |
+| `maxZoom`             | `max-zoom`              | The max zoom level available in the map. _Note: 20 is the best value to be used, only lower values are allowed. Is highly recommended to no change this value if you are not sure about the `maxZoom` supported by the map._ | `number`                                | `RECOMMENDED_MAX_ZOOM` |
+| `watchPosition`       | `watch-position`        | Indicates if the current location of the device is displayed on the map.                                                                                                                                                     | `boolean`                               | `false`                |
+| `zoom`                | `zoom`                  | The initial zoom level in the map.                                                                                                                                                                                           | `number`                                | `1`                    |
 
 ## Events
```

Comment:
The new event `userLocationChange` doesn't appear in the readme file.

Reply:
Fixed in 298d460